tests/test_utils.py

```diff
@@ -456,7 +449,9 @@ def include_paths_env_variable(self, other_directory: str) -> Generator:
         os.environ.pop("SNOWCLI_INCLUDE_PATHS")
 
     @staticmethod
-    def create_file(filepath: str, contents: List[str]) -> None:
-        with open(filepath, "w") as new_file:
-            for line in contents:
-                new_file.write(line + "\n")
+    def create_file(suffix: str, dir: str, contents: List[str]) -> str:
+        with tempfile.NamedTemporaryFile(suffix=suffix, dir=dir, delete=False) as tmp:
+            with open(tmp.name, "w") as new_file:
+                for line in contents:
+                    new_file.write(line + "\n")
+        return tmp.name
```

Comment:
Does it return a path or only a name?

If it's only a name, then you won't know the full path when using functions like `file_in_a_subdir` which calls `create_file`.

Reply:
Contrary to Path.name (that returns only filename) tempfile .name returns an absolute path